ui/lib/core/addon/components/form-field.hbs

```diff
@@ -233,6 +233,12 @@
         />
       {{/if}}
     </Toggle>
+  {{else if (eq @attr.options.editType "toggle")}}
```

Comment:
Have you updated the sidebranch recently? It's probably worth rebasing, Dan recently added this edit type to form fields: https://github.com/hashicorp/vault/pull/29820/files#diff-d10b01999845cdbb2bb7704e40fc343061f6d2a35d0269fc97fa684d0a2de66a